docs/manifest/schema.rst

```diff
@@ -0,0 +1,90 @@
+
+Manifest Schema
+===============
+
+.. code-block:: yaml
+
+    version: 1 # <integer> schema version for manifest
+               # required
+
+    type:  # type of script: easypy
+           # required
+
+    arguments:  # arguments for the script (optional)
+
+        # command line arguments may override the arguments provided here or in the profile
+
+        <key>: <value>  # if the key does not start with "-", "--" will be prepended to the argument string, e.g. --key value
+
+        # argparse 'nargs' using 'list'
+        key:  # values may be specified as a list, this will translate to "--key val1 val2"
```

Comment:
i think this schema will be somewhat more difficult to read.

it contains a lot of critical information

maybe worth reformatting and or building a quick reference/rule set